kbdd-example/src/test/kotlin/ru/fix/kbdd/example/cases/documentation/CookieRestTest.kt

```diff
@@ -0,0 +1,54 @@
+package ru.fix.kbdd.example.cases.documentation
+
+import io.qameta.allure.Epic
+import io.qameta.allure.Feature
+import org.junit.jupiter.api.Test
+import org.koin.core.KoinComponent
+import org.koin.core.inject
+import ru.fix.corounit.allure.Package
+import ru.fix.corounit.allure.invoke
+import ru.fix.kbdd.asserts.*
+import ru.fix.kbdd.example.MockServer
+import ru.fix.kbdd.example.TestFramework.makeCodeSnippet
+import ru.fix.kbdd.rest.Rest
+
+@Epic("Rest")
+@Feature("Cookie")
+@Package("Documentation")
+class CookieRestTest : KoinComponent {
+    val mockServer by inject<MockServer>()
+
+    @Test
+    suspend fun `sending and receiving elements of HTTP response`() {
```

Comment:
this test is about reading cookies from response, could you please name test method appropriately